src/SFA.DAS.RoATPService.Application/Validators/OrganisationValidator.cs

```diff
@@ -96,7 +89,7 @@ public bool IsValidStatus(OrganisationStatus status)
 
         public bool IsValidStatusId(int statusId)
         {
-            return (statusId >= 0 && statusId <= 2);
+            return (statusId >= 0 && statusId <= 3);
```

Comment:
Maybe as part of tech debt refactor this so checks to see if any of the valid values, i.e. 
if (statusId == OrganisationStatus.Active || statusId == OrganisationStatus.Onboarding, ..... etc